GVRf/Framework/jni/shaders/material/unlit_shader.cpp

```diff
@@ -98,14 +97,15 @@ void UnlitShader::render(const glm::mat4& mvp_matrix,
     glUseProgram(program_->id());
 
     glUniformMatrix4fv(u_mvp_, 1, GL_FALSE, glm::value_ptr(mvp_matrix));
-    glActiveTexture(GL_TEXTURE0);
+    glActiveTexture (GL_TEXTURE0);
```

Comment:
that seems to have been introduced by the formatter.  perhaps my settings are off.  if i remove that space, and then hit crtl-shift-f, it re-introduces that space.  I can remove it though.